src/components/Buttons/FutureButton/FutureButton.tsx

```diff
@@ -39,16 +42,16 @@ const FutureButton: React.FC<FutureButtonProps> = ({
 
 	return (
 		<button
-			className={`${styles.futureButton} ${isSelected ? styles.selected : ''} ${
-				glow ? styles.glow : ''
-			} ${loading ? styles.Loading : ''} ${alt ? styles.Alt : ''} ${
-				className ? className : ''
-			}`}
+			className={`${styles.futureButton} ${
+				glow ? ' ' : styles.futureButtonUnActive
+			} ${isSelected ? styles.selected : ''} ${glow ? styles.glow : ''} ${
+				loading ? styles.Loading : ''
+			} ${alt ? styles.Alt : ''} ${className ? className : ' '}`}
 			onMouseEnter={handleHover}
 			onMouseUp={handleClick}
 			style={style}
 		>
-			<div className={styles.content}>
+			<div className={`${styles.Content}`}>
```

Comment:
Minor: If it's just one class you don't need to wrap it like `'${styles.Content}'`, you can just do `className={styles.Content}`

Reply:
Yeah, I actually know this, I think I just missed it.